common/traits/druid_forms.txt

```diff
@@ -60,8 +60,8 @@ pack_form = {
 	martial = 4
 	diplomacy = -5
 	command_modifier = {
-		damage = 0.3
-		morale_offence = 0.3
+		damage = 0.25
+		morale_offence = 0.25
```

Comment:
I would decrease Pack Form buff to `0.15` for both modifiers. That is unless this form counts as a leadership trait, which I don't think it does

Reply:
pack form is supposed to be broken and when made avaible will be only accesible by a few chars